my-app/src/components/Button.tsx

```diff
@@ -0,0 +1,130 @@
+import React from 'react';
+import styled from '@emotion/styled';
+
+interface ThemeProps {
+	backgroundColor: string;
+	color: string;
+	border?: string;
+}
+
+interface SizeProps {
+	padding: string;
+	fontSize: string;
+	responsive?: {
+		small: {
```

Comment:
이부분 타이핑이 중복되는데, 반응형 사이즈 인터페이스도 하나 만들어두면 어떨까요?

```suggestion
		interface ResponsiveSize {
		  padding?: string;
		  fontaine?: string
		}
```

Reply:
감사합니다 수정했습니다!